<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>skanderwindow</class>
 <widget class="QMainWindow" name="skanderwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="TabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1091</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="TabWidgetPage1">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>275</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>num_facture: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_num_facture"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>montant: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_montant"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>169</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>type :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>client</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>fournisseur</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>date : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_montant_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>240</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tab_fournisseur">
      <property name="geometry">
       <rect>
        <x>635</x>
        <y>20</y>
        <width>301</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tab_produit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>301</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tab_client">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>290</y>
        <width>621</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>271</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>produit: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_produit"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="afficher">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Supprimer">
      <property name="geometry">
       <rect>
        <x>52</x>
        <y>350</y>
        <width>146</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>52</x>
        <y>320</y>
        <width>82</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>num_facture: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_num_facture_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>56</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="TabWidgetPage4">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>281</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bp_tricroi">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>tri croi</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bp_tridecroi">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>50</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>tri decroi</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bp_rechercher">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>240</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rech">
      <property name="geometry">
       <rect>
        <x>312</x>
        <y>210</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>170</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>imprimer pdf</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>110</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>statistiques</string>
      </property>
     </widget>
     <widget class="QPushButton" name="excel">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>140</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Excel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="calcul">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>80</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>calcul</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>221</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>num_facture </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_num_facture_3"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>produit: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_produit_2"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>montant: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_montant_2"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>460</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>type :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>390</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>client</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_montant_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>430</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>410</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>fournisseur</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>date : </string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_produit_3">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>0</y>
        <width>301</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tab_client_3">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>250</y>
        <width>621</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tab_fournisseur_3">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>0</y>
        <width>301</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="verifier">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>450</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Vérifier</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1144</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
